src/Digdir.Domain.Dialogporten.Infrastructure/InfrastructureExtensions.cs

```diff
@@ -74,15 +74,19 @@ public static class InfrastructureExtensions
 
         services.ConfigureFusionCache(nameof(Altinn.NameRegistry), new()
         {
-            Duration = TimeSpan.FromDays(1)
+            Duration = TimeSpan.FromHours(24),
+            FailSafeMaxDuration = TimeSpan.FromHours(26)
```

Comment:
(off-topic) Added to avoid Fusion cache complaining about Duration being larger than FailSafeMaxDuration